### PR TITLE
[ACS-4277] Changed whitespace to normal in order to make longer notifications messages visible

### PR DESCRIPTION
**Pull request by arohilaGL**
…essage visible.

**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [x] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)
Notification messages get truncated.

**What is the new behaviour?**
We don't want to limit the visibility of texts in order to make it convenient for accessibility reasons. 

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by nikita-web-ua** [comment]
Typo in PR title + missing space after square bracket

File: lib/core/src/lib/notifications/components/notification-history.component.scss [hunk 1]
```diff
@@ -69,3 +69,7 @@
         padding: 0;
     }
 }
+
+.mat-list-base .mat-list-item .mat-line {
```
Comment (mauriziovitale) [inline lib/core/src/lib/notifications/components/notification-history.component.scss:73]: what do exactly mean with longer? this css seems to be changing .mat-list-item globally are we sure we don't have unexpected impacts on the apps?

**Comment by DenysVuika** [comment]
you have multiple typos in the PR title, `whitespace` is a single word, and not `white space`

File: lib/core/src/lib/notifications/components/notification-history.component.scss (hunk 1 above)
Comment (DenysVuika) [reply lib/core/src/lib/notifications/components/notification-history.component.scss:73]: those styles should be scoped to the notification history, not being global ones, also as @mauriziovitale said, what does this change do? can you add some screenshots or explanation?

**Comment by DenysVuika** [comment]
@arohilaGL please put the spaces after jira ticket, and pay attention to the comments

File: lib/core/src/lib/notifications/components/notification-history.component.scss (hunk 1 above)
Comment (arohilaGL) [reply lib/core/src/lib/notifications/components/notification-history.component.scss]: scoped the change to notification history. Screenshots for your reference are present https://alfresco.atlassian.net/browse/ACS-4277.

**Review by DenysVuika — APPROVED** [review]

File: lib/core/src/lib/notifications/components/notification-history.component.scss [hunk 2]
```diff
@@ -69,3 +69,8 @@
         padding: 0;
     }
 }
+
+.mat-list-base .mat-list-item.adf-notification-history-menu-no-message .mat-line,
+.mat-list-base .mat-list-item .adf-notification-history-menu-message.mat-line {
```
Comment (nikita-web-ua) [inline lib/core/src/lib/notifications/components/notification-history.component.scss:74]:
```suggestion
.mat-list-base .mat-list-item .adf-notification-history-menu-no-message .mat-line,
.mat-list-base .mat-list-item .adf-notification-history-menu-message .mat-line {
```

File: lib/core/src/lib/notifications/components/notification-history.component.scss [hunk 3]
```diff
@@ -69,3 +69,8 @@
         padding: 0;
     }
 }
+
+.mat-list-base .mat-list-item.adf-notification-history-menu-no-message .mat-line,
```
Comment (AleksanderSklorz) [inline lib/core/src/lib/notifications/components/notification-history.component.scss:73]: Small comment but that can be a bit shorter: 
```scss
.mat-list-base .mat-list-item {
  &.adf-notification-history-menu-no-message .mat-line, .adf-notification-history-menu-message.mat-line {
    white-space: normal;
  }
}
```

Thanks that you don't need to repeat .mat-list-base .mat-list-item twice.

**Review by AleksanderSklorz — APPROVED** [review]

File: lib/core/src/lib/notifications/components/notification-history.component.scss (hunk 2 above)
Comment (arohilaGL) [reply lib/core/src/lib/notifications/components/notification-history.component.scss:74]: These is how it is supposed to be, this is the selector of where I need to make the change

Comment (AleksanderSklorz) [reply lib/core/src/lib/notifications/components/notification-history.component.scss:74]: @nikita-web-ua please have a look at my comments here in Jatin's PR where you commented space for selectors too. I explained there why sometimes we need space and sometimes no for selectors :) 
https://github.com/Alfresco/alfresco-content-app/pull/3018#discussion_r1118517484

**Comment by mauriziovitale** [comment]
I don't think we should... the PR is changing the general mat-list-item. Therefore  all the apps will be impacted. Please work with alfresco BDU to make sure there won't be any side effects.

**Review by MichalKinas — APPROVED** [review]

File: lib/core/src/lib/notifications/components/notification-history.component.scss (hunk 3 above)
Comment (DenysVuika) [reply lib/core/src/lib/notifications/components/notification-history.component.scss:73]: the path for the css is unreasonably long, just create a dedicated class for the mat-line in the Notification component, and style it directly, i.e. 

```css
.adf-notification-message-line { white-space: normal; }
```

**Review by DenysVuika — CHANGES_REQUESTED** [review]

**Pull request closed without merging**